### PR TITLE
Initial implementation of the Converter between SDK structures and Protobuf Structures.

### PR DESCRIPTION
**Pull request by humdrum**
**What this PR does / why we need it**:
Initial implementation of the Converter between SDK structures and Protobuf Structures.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**: 

```release-note
NONE
```

**Additional documentation**:

```docs

```

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

File: Sources/API/Converter.swift [hunk 1]
```diff
@@ -48,4 +48,990 @@ enum Converter {
...
+    /**
+     * `toChangePack` converts the given model to Protobuf format.
+     */
+    static func toChangePack(pack: ChangePack) -> PbChangePack {
+        var pbChangePack = PbChangePack();
+        pbChangePack.documentKey = pack.getDocumentKey()
+        pbChangePack.checkpoint = toCheckpoint(pack.getCheckpoint())
+        pbChangePack.changes = toChanges(pack.getChanges())
+        // TODO: snapshot can be nil!
+        pbChangePack.snapshot = pack.getSnapshot() ?? Data()
```
Comment (humdrum) [inline Sources/API/Converter.swift:953]: 'snapshot' of ChangePack is optional, but PbChangePack is required. 
We need the default value of snapshot or change PbChangePack's to optional. 

@hackerwins

File: Sources/API/Converter.swift [hunk 2]
```diff
@@ -48,4 +48,990 @@ enum Converter {
...
+// MARK: Change
+extension Converter {
+    /**
+     * `toChange` converts the given model to Protobuf format.
+     */
+    static func toChange(_ change: Change) -> PbChange {
+        var pbChange = PbChange();
+        pbChange.id = toChangeID(change.getID())
+        // TODO: message can be nil!
+        pbChange.message = change.getMessage() ?? ""
```
Comment (humdrum) [inline Sources/API/Converter.swift:984]: 'message' of Change is optional, but PbChange's is required. 
We need the default value of snapshot or change PbChange's to optional. 

@hackerwins

File: Sources/Document/Change/Checkpoint.swift [hunk 3]
```diff
@@ -28,17 +28,17 @@ struct Checkpoint: Equatable {
      */
     static let initial = Checkpoint(serverSeq: 0, clientSeq: 0)
     private var serverSeq: Int64
-    private var clientSeq: Int
+    private var clientSeq: UInt32
 
-    init(serverSeq: Int64, clientSeq: Int) {
+    init(serverSeq: Int64, clientSeq: UInt32) {
```
Comment (hyeongsik-won) [inline Sources/Document/Change/Checkpoint.swift:33]: Is serverSeq's type different from clientSeq' type?

File: Tests/API/V1/ConverterTests.swift [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+        let addOperation = AddOperation(parentCreatedAt: TimeTicket.initial,
+                                        previousCreatedAt: TimeTicket.initial,
+                                        value: Primitive(value: .null, createdAt: TimeTicket.initial),
+                                        executedAt: TimeTicket.initial)
+        let change = Change(id: ChangeID.initial, operations: [addOperation], message: "AddOperation")
+        do {
+            let converted = try Converter.fromChanges([Converter.toChange(change)]).first
+            XCTAssertEqual(change.getStructureAsString(), converted?.getStructureAsString())
+        } catch {
+            XCTAssert(false, "\(error)")
```
Comment (hyeongsik-won) [inline Tests/API/V1/ConverterTests.swift:99]: What about using XCTFail?
XCTFail is easy to see.

**Review by hyeongsik-won — COMMENTED** [review]
Thank you for your contribution.

File: Sources/API/GRPCTypeAlias.swift [hunk 5]
```diff
@@ -18,9 +18,25 @@
...
 import Foundation
 
-typealias ValueType = Yorkie_V1_ValueType
 typealias YorkieServiceNIOClient = Yorkie_V1_YorkieServiceNIOClient
 typealias ActivateClientRequest = Yorkie_V1_ActivateClientRequest
 typealias DeactivateClientRequest = Yorkie_V1_DeactivateClientRequest
 typealias YorkieServiceAsyncClient = Yorkie_V1_YorkieServiceAsyncClient
+typealias AttachDocumentRequest = Yorkie_V1_AttachDocumentRequest
 
+typealias PbChange = Yorkie_V1_Change
```
Comment (hyeongsik-won) [inline Sources/API/GRPCTypeAlias.swift:27]: How about making Pb as PB?

File: Sources/API/Converter.swift [hunk 6]
```diff
@@ -48,4 +48,985 @@ enum Converter {
...
+
+extension String {
+    // Same as toUint8Array in JS
+    var toData: Data? {
+        guard self.count % 2 == 0 else {
+            return nil
+        }
+                
+        var data = Data()
+        for i in stride(from: 0, to: self.count, by: 2) {
```
Comment (hyeongsik-won) [inline Sources/API/Converter.swift:1020]: How about making ```i``` as a word?

File: Sources/Document/Change/Checkpoint.swift (hunk 3 above)
Comment (humdrum) [reply Sources/Document/Change/Checkpoint.swift:33]: Yes. see the protobuf message below.

https://github.com/yorkie-team/yorkie-ios-sdk/blob/7084ee76600d7939ea99b867ebe663e536a7eb5c/Sources/API/V1/Protos/resources.proto#L275-L278

**Review by hyeongsik-won — APPROVED** [review]
LGTM!

File: Sources/API/Converter.swift (hunk 1 above)
Comment (humdrum) [reply Sources/API/Converter.swift:947]: Handle empty bytes as nil.

File: Sources/API/Converter.swift (hunk 2 above)
Comment (humdrum) [reply Sources/API/Converter.swift:977]: Handle empty string as nil.

**Review by hackerwins — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**